susemanager/susemanager.spec

```diff
@@ -188,7 +192,7 @@ for i in `find . -type f -not -name 'mgr-salt-ssh'`;
 do
     sed -i '1s=^#!/usr/bin/\(python\|env python\)[0-9.]*=#!/usr/bin/python3=' $i;
```

Comment:
No need to change this?

Reply:
If my commentse didn't make any sense, go ahead without getting an approval from my side :-)

Reply:
I think it's done in two steps. 
the  first sed (the one you marked here), sets python2 or python3.
The one in the other line does the set to  python3.11 only  for mgr-salt-ssh